sdk/search/azure-search-documents/azure/search/documents/_service/_utils.py

```diff
@@ -151,3 +159,28 @@ def listize_synonyms(synonym_map):
     # type: (dict) -> dict
     synonym_map["synonyms"] = synonym_map["synonyms"].split("\n")
     return synonym_map
+
+def get_access_conditions(model, match_condition=MatchConditions.Unconditionally):
```

Comment:
Are you missing tests for this function?

Reply:
added several *_if_unchanged tests where i specify match conditions

Reply:
also added test to raise when a string is passed along with match condition